dev/source/docs/ros2-gazebo.rst

```diff
@@ -32,6 +32,20 @@ Update ROS dependencies:
     sudo apt update
     rosdep update
     rosdep install --from-paths src --ignore-src -r
+    # Ignore the rosdep warnings at the end if any
+
+Installing build dependencies:
```

Comment:
Move this to the `ros2` page, that's where you are supposed to do it.

Reply:
Ah I see. 
I will update it over there